README.rst

```diff
@@ -0,0 +1,123 @@
+=========================
+nRF70 BM Driver README
+=========================
+
+This document provides instructions on how to set up and use the nRF70 BM driver with `west` for nRF boards or `git submodule` for customer boards.
+
+
+Setup Instructions
+==================
+
+Using `west` for nRF Boards
+---------------------------
+
+`west` is the official Zephyr Project meta-tool that manages the Zephyr Project source code and its dependencies.
+It is recommended to use `west` to build and flash the nRF70 BM driver for nRF boards.
+
+1. **Install West:**
+
+    Follow the instructions on the official Zephyr Project documentation to install `west`:
+    https://docs.zephyrproject.org/latest/guides/west/install.html
+
+2. **Clone the Repository:**
+
+    Clone the main repository using `west`:
+
+    ```sh
+    west init -m https://github.com/nrfconnect/nrf70-bm
+    cd nrf70-bm-driver
+    west update
+    ```
+
+3. **Build the Project:**
+
+    Use `west` to build the project for your specific board:
+
+    ```sh
+    west build -b your_board_name
+    ```
+
+4. **Flash the Board:**
+
+    Flash the firmware onto your nRF board:
+
+    ```sh
+    west flash
+    ```
+
+Using `git submodule` for Customer Boards
+------------------------------------------
+
+1. **Clone the Repository:**
+
+    Clone the main repository and its submodules:
+
+    ```sh
+    git clone https://github.com/nrfconnect/nrf70-bm --recurse-submodules
+    ```
+
+    If you have already cloned the repository without submodules, you can initialize and update the submodules with:
+
+    ```sh
+    git submodule update --init --recursive
+    ```
+
+2. Build and flashing instructions depend on the customer board and the build system used.
+
+
+Directory Structure
+===================
+
+The directory structure should look like this:
+
+.. code-block:: none
+
+    nrf70_bm_lib/         # nRF70 BM driver library
+    sdk_nrfxlib/          # nRF Connect SDK nrfxlib
+      nrf_wifi/           # nRF70 Wi-Fi OS agnostic driver
+    samples/              # Sample applications
+      radio_test_bm/      # Radio test sample application
+      scan_bm/            # Scan sample application
+    nrf70_zephyr_shim/    # Zephyr shim for nRF70, reference for customer boards
+
+
+Porting guide
+=============
+
+The nRF70 BM library is designed to be portable across different platforms.
```

Comment:
```suggestion
The nRF70 Series BM Driver is designed to be portable across different platforms.
```

I assume that the BM Library is - by definition - portable. But the Library is only a part of the BM driver.